.github/workflows/ci.yaml

```diff
@@ -34,13 +34,16 @@ jobs:
       - name: Install testing scripts
         run: nix-env -f default.nix -iA tests withTools
 
-      - name: Run coverage (IO tests and Spec tests against PostgreSQL 13)
+      - name: Run coverage (IO tests and Spec tests against PostgreSQL 14)
         run: postgrest-coverage
       - name: Upload coverage to codecov
         uses: codecov/codecov-action@v2
         with:
           files: ./coverage/codecov.json
 
+      - name: Run the spec tests against PostgreSQL 13
+        if: always()
+        run: postgrest-with-postgresql-13 postgrest-test-spec
```

Comment:
@monacoremo Running the command `postgrest-coverage` already uses the PostgreSQL 14 in my local environment, does it do the same for the CI or is anything else needed? If so, the missing one would be the tests on v13, which I added here.